GitUI/Script/FilePrompt.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Linq;
+using System.Windows.Forms;
+
+namespace GitUI.Script
+{
+    public partial class FilePrompt : Form
+    {
+        public string FileInput { get; private set; } = string.Empty;
+
+        public FilePrompt()
+        {
+            InitializeComponent();
+        }
+
+        protected override void OnShown(EventArgs e)
+        {
+            txt_FilePath.Focus();
+        }
+
+        private void Btn_OK_Click(object sender, EventArgs e)
+        {
+            if (!string.IsNullOrEmpty(txt_FilePath.Text))
+            {
+                FileInput = txt_FilePath.Text;
+                DialogResult = DialogResult.OK;
+            }
+
+            Close();
+        }
+
+        private void Btn_Browse_Click(object sender, EventArgs e)
+        {
+            const string separator = " ";
+            using (var browseDialog = new OpenFileDialog
+            {
+                Multiselect = true,
+                InitialDirectory = "."
```

Comment:
Suggest to add:

```
                AutoUpgradeEnabled = true,
                CheckFileExists = true,
                CheckPathExists = true,
                RestoreDirectory = true, 
                SupportMultiDottedExtensions = true,
                ValidateNames = true
```

Some of them are probably `true` by default, but it is easier to be explicit.

Reply:
I can agree with that suggestion. But I have some questions::
About `RestoreDirectory` - the official documentation says:
> true if the dialog box restores the current directory to the previously selected directory if the user changed the directory while searching for files; otherwise, false. The default value is false.

What value should I give it?
Same for `SupportMultiDottedExtensions`:
> true if the dialog box supports multiple file name extensions; otherwise, false. The default is false.